### PR TITLE
[WIP] moving SwiftmailerBundle docs to bundle repo

### PR DESCRIPTION
**Pull request by snoek09**
Added installation chapter.

Easy picks on bundle side from https://github.com/symfony/symfony-docs/issues/4983

### CODE REVIEW

**Comment by xabbuh** [comment]
:+1:

File: Resources/doc/index.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+Step 1: Download the Bundle
+---------------------------
+
+Open a command console, enter your project directory and execute the
+following command to download the latest stable version of this bundle:
+
+```bash
+$ composer require symfony/swiftmailer-bundle "~2.3"
+```
```
Comment (WouterJ) [inline Resources/doc/index.rst]: this is markdown syntax, but the docs have to be written in rst in order to be rendered on symfony.com. This means you have to change this to:

```rst
.. code-block:: bash

    $ composer require symfony/swiftmailer-bundle "~2.3"
```

File: Resources/doc/index.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+This command requires you to have Composer installed globally, as explained
+in the [installation chapter](https://getcomposer.org/doc/00-intro.md)
```
Comment (WouterJ) [inline Resources/doc/index.rst]: and for the same reason, this should be:

```rst
in the `installation chapter`_ of the Composer documentation.
```

With this at the end of the file:

```rst
.. _`installation chapter`: https://getcomposer.org/doc/00-intro.md
```

File: Resources/doc/index.rst [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+of the Composer documentation.
+
+Step 2: Enable the Bundle
+-------------------------
+
+Then, enable the bundle by adding the following line in the `app/AppKernel.php`
```
Comment (WouterJ) [inline Resources/doc/index.rst]: then these single backticks have to be double ones

File: Resources/doc/index.rst [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+            new Symfony\Bundle\SwiftmailerBundle\SwiftmailerBundle(),
+        );
+
+        // ...
+    }
+
+    // ...
+}
+```
```
Comment (WouterJ) [inline Resources/doc/index.rst]: and this code block has to be changed the same way as the other code block (but then using PHP as highlighting language)

**Comment by fabpot** [comment]
Thank you @snoek09.

**Pull request closed without merging**